### ISSUE TITLE
Restore account from eos-voter 0.3.3

### ISSUE THREAD

**Issue by miclaus** [greymass/anchor#1069]
I'm trying to restore an old account which I had setup with eos-voter version 0.3.3

Version 0.3.3 did not have a "Save Backup" button as mentioned by https://greymass.com/en/blog/migrating-to-anchor-from-eos-voter-greymass-wallet/

I do have a copy of `~/Library/Application Support/eos-voter/config.json`, but I'm not sure what to do with it now.

Could you maybe guide me on how to backup & restore the account with the newest version of Anchor?

Thank you!

**Comment by miclaus** [comment]
To be more clear I have eos-voter 0.3.3 installed with the old account and I can still "Submit votes for selected producers".

**Comment by aaroncox** [comment]
Hah, quite the legacy version! 

First question would be, do you have your private keys backed up somewhere external of the wallet? If so, the easiest path would probably be to just install Anchor and import those keys directly without going through the upgrade process. 

If you do not have them backed up elsewhere (which I'll always recommend), then I'll recommend saving a copy of that `config.json` file you referenced someplace safe (as a backup in the event something screws up during the upgrade), and upgrade to eos-voter 0.7.12, which you can find here: 

https://github.com/greymass/anchor/releases/tag/v0.7.12

This version will have the backup feature that 0.3.3 is missing and allow you to go through the upgrade process using a backup.

**Comment by miclaus** [comment]
Hah, yes! It was the one tool I trusted for voting back in the days 😄 

Unfortunatelly, I somehow don't have the private keys backed up (or I have them somewhere buried deep). I know I should though.

I downloaded version 0.7.12 and managed to use the backup feature. I then used the "Import Backup" feature from the initial screen and there was my account. It then asked me to Upgrade the account, so I did. I then refreshed my votes to check whether everything was still working. Seems that it is!

Now I can even backup my private keys.

Thanks a bunch for the help!
